src/go/flatgeobuf_test.go

```diff
@@ -77,6 +77,9 @@ func TestCreateFGBFileAndBasicSearch(t *testing.T) {
 				createSquareFeature(-1, 0, 0, 1, 2),
 				createSquareFeature(-1, -1, 0, 0, 3),
 				createSquareFeature(0, -1, 1, 0, 4),
+				createMultiPolygonFeature([][]float64{
```

Comment:
Would you mind moving this to its own test? It seems important enough to deserve this (also it might not match the basic test here).

Reply:
Good point, I created another test

Reply:
Thanks.